internal/cli/profile_check.go

```diff
@@ -32,25 +33,44 @@ var (
 		RunE:  runProfileCheck,
 	}
 
-	flagProfileCheckName      string
-	flagProfileCheckPath      string
-	flagProfileCheckOsVersion string
+	flagProfileCheckName       string
+	flagProfileCheckPath       string
+	flagProfileCheckRemoteOnly string
+	flagProfileCheckOsVersion  string
 )
 
 func init() {
 	cmdProfile.AddCommand(cmdProfileCheck)
 	cmdProfileCheck.Flags().StringVar(&flagProfileCheckName, "name", "", "profile name to check")
 	cmdProfileCheck.Flags().StringVar(&flagProfileCheckPath, "file", "", "profile file to check")
+	cmdProfileCheck.Flags().StringVar(&flagProfileCheckRemoteOnly, "remote-only", "", "whether to only check addons with an explicit remote")
 	cmdProfileCheck.Flags().StringVarP(&flagProfileCheckOsVersion, "os-release", "n", "", "override OS version")
 }
 
+func parseFlagRemoteOnly() bool {
+	remoteOnly := false
+	env, ok := viper.Get("CHECK_REMOTE_ONLY").(string)
+	if ok && env != "" {
+		if value, err := strconv.ParseBool(env); err == nil {
+			remoteOnly = value
+		}
+	}
+	if flagProfileCheckRemoteOnly != "" {
+		if value, err := strconv.ParseBool(env); err == nil {
```

Comment:
Shouldn't this be `flagProfileCheckRemoteOnly`?

Reply:
Indeed